### PR TITLE
ACM#11171: Part 1: Update user-replaced values

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-11171

### CODE REVIEW

File: clusters/hosted_control_planes/access_hosted_cluster.adoc [hunk 1]
```diff
@@ -7,14 +7,14 @@ You can access the hosted cluster by either getting the `kubeconfig` file and `k
 +
 The secret name formats are as follows:
 
-** `kubeconfig` secret: `<hostingNamespace>-<name>-admin-kubeconfig` (clusters-hypershift-demo-admin-kubeconfig)
-** `kubeadmin` password secret: `<hostingNamespace>-<name>-kubeadmin-password` (clusters-hypershift-demo-kubeadmin-password)
+** `kubeconfig` secret: `<hosting-cluster-namespace>-<name>-admin-kubeconfig` (clusters-hypershift-demo-admin-kubeconfig)
+** `kubeadmin` password secret: `<hosting-cluster-namespace>-<name>-kubeadmin-password` (clusters-hypershift-demo-kubeadmin-password)
```
Comment (jparrill) [inline clusters/hosted_control_planes/access_hosted_cluster.adoc]: Replace `hosting` by `hosted`

File: clusters/hosted_control_planes/config_ingress_dns_kubevirt.adoc [hunk 2]
```diff
@@ -45,21 +45,29 @@ export WORKER_COUNT="2"
...
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \/// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--base-domain <basedomain> <6>
 ----
 +
-<1> The name of the hosted cluster, which for example purposes, is `example`.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_ingress_dns_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/config_ingress_dns_kubevirt.adoc [hunk 3]
```diff
@@ -93,34 +101,43 @@ See the following example output:
...
+ingress                                    4.x.0     True        False         True       28m     The "default" ingress controller reports Degraded=True: DegradedConditions: One or more other status conditions indicate a degraded state: CanaryChecksSucceeding=False (CanaryChecksRepetitiveFailures: Canary route checks for the default ingress controller are failing)
 ----
 +
 Replace `4.x.0` with the supported {ocp-short} version that you want to use.
 
 +
 The next steps fixes the errors in the output.
 +
-*Note:* If your cluster is on bare metal, you might need MetalLB so that you can set up load balancer services. For more information, see _Optional: Configuring MetalLB_.
+*Note:* If your cluster is on bare metal, you might need MetalLB to set up load balancer services. For more information, see _Optional: Configuring MetalLB_.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_ingress_dns_kubevirt.adoc]: when you say "If your cluster is on bare metal", what cluster are you referring to? Management Cluster or Hosted cluster? I think is worth to be specific.

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 4]
```diff
@@ -6,54 +6,69 @@ If no advanced configuration is provided, the default storage class is used for
...
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--infra-storage-class-mapping=<storage-class>/<hosted-storage-class> \// <6>
 ----
-
-After you create the hosted cluster, the `guest-sc1` and `guest-sc2` storage classes are visible within the hosted cluster. When you create a PVC within the hosted cluster that uses one of those storage classes, KubeVirt CSI provisions that volume by using the infrastructure storage class mapping that you configured during cluster creation.
++
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 5]
```diff
@@ -6,54 +6,69 @@ If no advanced configuration is provided, the default storage class is used for
...
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--root-volume-storage-class <> \// <6>
+--root-volume-size <volume-size> <7>
 ----
++
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 6]
```diff
@@ -63,53 +78,64 @@ KubeVirt image caching is an advanced feature that you can use to optimize both
...
---root-volume-cache-strategy=PVC
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--root-volume-cache-strategy=PVC <6>
 ----
++
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/config_storage_kubevirt.adoc [hunk 7]
```diff
@@ -63,53 +78,64 @@ KubeVirt image caching is an advanced feature that you can use to optimize both
...
---etcd-storage-class=${ETCD_STORAGE}
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--etcd-storage-class=<etcd-storage-class-name> <6>
 ----
++
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/config_storage_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_cluster_bm.adoc [hunk 8]
```diff
@@ -3,44 +3,54 @@
...
     --control-plane-availability-policy SingleReplica \
-    --release-image=quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE} \
-    --api-server-address=api.${HOSTED_CLUSTER_NAME}.${BASEDOMAIN}
+    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp-release> <8>
 ----
 
 +
-<1> Required by the agent cluster API provider and must already exist. Use this as the `--agent-namespace` flag in the `hcp create cluster` command. Ensure that agents are available in this namespace. You can find the list of agents by using the `oc get agent -n ${HOSTED_CONTROL_PLANE_NAMESPACE}` command.
-<2> Replace `4.x.0` with the supported version that you want to use.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_cluster_bm.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_cluster_kubevirt.adoc [hunk 9]
```diff
@@ -10,27 +10,30 @@ With {ocp-short} 4.14 and later, you can create a cluster with KubeVirt, to incl
...
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--etcd-storage-class=<etcd-storage-class> <6>
 ----
 +
-Replace values where needed.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_cluster_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_cluster_kubevirt.adoc [hunk 10]
```diff
@@ -109,21 +112,29 @@ You can create a hosted cluster by using the `hcp` command line interface.
...
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas <worker-count> \// <2>
+--pull-secret <path-to-pull-secret> \// <3>
+--memory <value-for-memory> \// <4>
+--cores <value-for-cpu> \// <5>
+--infra-namespace=<hosted-cluster-namespace>-<hosted-cluster-name> \// <6>
+--infra-kubeconfig-file=<path-to-external-infra-kubeconfig> <7>
 ----
++
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_cluster_kubevirt.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_cluster_non_bm.adoc [hunk 11]
```diff
@@ -3,44 +3,54 @@
...
     --control-plane-availability-policy SingleReplica \
-    --release-image=quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE} \
-    --api-server-address=api.${HOSTED_CLUSTER_NAME}.${BASEDOMAIN}
+    --release-image=quay.io/openshift-release-dev/ocp-release:<ocp-release> <8>
 ----
 
 +
-<1> Required by the agent cluster API provider and must already exist. Use this as the `--agent-namespace` flag in the `hcp create cluster` command. Ensure that agents are available in this namespace. You can find the list of agents by using the `oc get agent -n ${HOSTED_CONTROL_PLANE_NAMESPACE}` command.
-<2> Replace `4.x.0` with the supported version that you want to use.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_cluster_non_bm.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_hosted_aws.adoc [hunk 12]
```diff
@@ -1,27 +1,26 @@
...
+--node-pool-replicas=<node-pool-replica-count> \// <2>
+--base-domain <basedomain> \// <3>
+--pull-secret <path-to-pull-secret> \// <4>
+--aws-creds <path-to-aws-credentials> \// <5>
+--region <region> \// <6>
+--zones <zones> <7>
 ----
 
-<1> The `--zones` flag must specify availability zones within the region, which is specified by the `--region` flag.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_hosted_aws.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_hosted_aws.adoc [hunk 13]
```diff
@@ -46,39 +45,40 @@ See the following example:
...
---region $REGION \
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas=<node-pool-replica-count> \// <2>
+--base-domain <basedomain> \// <3>
+--pull-secret <path-to-pull-secret> \// <4>
+--aws-creds <path-to-aws-credentials> \// <5>
+--region <region> <6>
 ----
 
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_hosted_aws.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/create_hosted_aws.adoc [hunk 14]
```diff
@@ -46,39 +45,40 @@ See the following example:
...
+hcp create cluster aws \
+--name <hosted-cluster-name> \// <1>
+--region <region> \// <2>
+--namespace <hosted-cluster-namespace> \// <3>
+--secret-creds <my-aws-cred> <4>
 ----
 
-<1> Because the secret contains the SSH keys, pull secret, base domain, and AWS credentials, you can use the `hcp create cluster aws` command with the `--secret-creds <my_aws_cred>` flag, where `<my_aws_cred>` is the cloud provider secret name. 
-<2> If the secret is not in the default `clusters` namespace, you must specify the namespace; for example, `--secret-creds <my_aws_cred> --namespace <name_of_namespace>`.
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/create_hosted_aws.adoc]: replace "for example, `example`" by "for instance, `example`"

File: clusters/hosted_control_planes/deploy_aws_private_clusters.adoc [hunk 15]
```diff
@@ -73,38 +73,41 @@ aws iam create-access-key --user-name=hypershift-operator
...
-<1> The `--endpoint-access` flag designates whether a cluster is public or private.
+--name <hosted-cluster-name> \// <1>
+--node-pool-replicas=<node-pool-replica-count> \// <2>
+--base-domain <basedomain> \// <3>
+--pull-secret <path-to-pull-secret> \// <4>
+--aws-creds <path-to-aws-credentials> \// <5>
+--region <region> \// <6>
+--endpoint-access Private <7>
+----
+<1> Specify the name of your hosted cluster, for example, `example`.
```
Comment (jparrill) [inline clusters/hosted_control_planes/deploy_aws_private_clusters.adoc]: replace "for example, `example`" by "for instance, `example`"

**Review by jparrill — CHANGES_REQUESTED** [review]
Dropped some comments

**Review by jparrill — APPROVED** [review]
/lgtm

**Review by swopebe — APPROVED** [review]
Thank you so much for addressing this.

**Pull request merged**